azkaban-common/src/main/java/azkaban/event/Event.java

```diff
@@ -26,7 +26,13 @@
     JOB_FINISHED,
     JOB_STATUS_CHANGED,
     EXTERNAL_FLOW_UPDATED,
-    EXTERNAL_JOB_UPDATED
+    EXTERNAL_JOB_UPDATED,
```

Comment:
Some comments on each kind of event would help. 
cosmetic: AZ_DOWNLOAD_FILE and USER_DOWNLOAD_FILE seemed a bit confusing? What files? Are we talking about project zips?